firestore-shorten-urls-bitly/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Version 0.1.2
+
+fixed - Migrates Bitly API to v4 since v3 will be deactivated on March 1, 2020 (issue #170).
```

Comment:
Do we want to add anything here to the CHANGELOG that says that they MUST update their extension to this version before March 1st or their extension will stop working?  We might add this warning someplace else, as well, but it should probably be here, too.

@karayu 
@laurenzlong

Reply:
yes good idea.

Reply:
How does this sound @rachelsaunders 

fixed - Migrates Bitly API to v4. Users must upgrade to this version before March 1, 2020 otherwise their extension will stop working. (issue #170).

Reply:
fixed - Migrates Bitly API to v4 since v3 will be deactivated on March 1, 2020 (Issue #170).

**Important:** Update your extension to at minimum v0.1.2 before March 1, 2020; otherwise, your installed extension will stop working.

Reply:
Fixed.